### DISCUSSION TITLE
Draft mode

### DISCUSSION THREAD

**Discussion by BerndDemian** [typst/typst#571] [Ideas]
Is in Typst a Draft mode possible, like in Latex?
Because my document has over 120 photos, the compiling will take 2 minutes, so little text change a very slow.
Thanks

**Comment by astrale-sharp** [comment]
As a workaround you could do something like 
```
#let image(..x) = box(width: default_width, ..x)
```
and comment it to go out of draft mode.

But no currently typst doesn't have a draft mode

**Reply by ararunaufc** [reply]
It's a good idea, but maybe not acceptable in some cases, because one of the advantages of LaTeX's draft mode is to show boxes of the exact size of the image in order to get to the final layout by saving some time.

**Comment by laurmaedje** [comment]
We can probably optimize that quite easily so that it is fast enough. Currently, encoding images for the PDF is not memoized and that is probably the bottleneck.

**Comment by BerndDemian** [comment]
Thanks for the answer and good luck!
